.travis.yml

```diff
@@ -29,7 +29,7 @@ install:
   - conda config --add channels morphman
 
   # Set up environment
-  - conda create -n morphman_env morphman pytest
+  - conda create -n morphman_env morphman pytest llvm=3.3
```

Comment:
We might need to pin this for the Linux distribution. I am experiencing errors when running the latest morphman version on Linux, which currently installs `llvm-10`.